Dockerfile

```diff
@@ -51,8 +51,8 @@ RUN --mount=type=cache,target=/opt/ccache \
 
 FROM conda as conda-installs
 ARG PYTHON_VERSION=3.8
-ARG CUDA_VERSION=11.0
-ARG CUDA_CHANNEL=defaults
+ARG CUDA_VERSION=11.1
+ARG CUDA_CHANNEL=nvidia
```

Comment:
Do we also to change CUDA_CHANNEL in docker.Makefile as well?

Reply:
AFAIK cudatoolkit=11.1 only exist in conda-forge

Reply:
> AFAIK cudatoolkit=11.1 only exist in conda-forge

It exists in the nvidia channel as well: https://anaconda.org/nvidia/cudatoolkit/files